packages/venia-ui/lib/components/HomePage/index.js

```diff
@@ -0,0 +1 @@
+export { default } from './homePage';
```

Comment:
The home page, being powered by an EE feature, needs to be "gated" through the use of `.ee.js` and `.ce.js` extensions.

We could possibly replace the current home page route (category?) with two files, one that directs to this "new home" experience for EE backend and one that uses the default category page for CE backends.

You can see how we do it for gift card summary in https://github.com/magento/pwa-studio/tree/develop/packages/venia-ui/lib/components/CartPage/PriceSummary

Reply:
@jimbo Hi! Could you pls clarify this one. Since PWA Studio 7.0 there is a new look for Home Page. As far as I can see, it is rendering "Home" CMS Page, created by EE Page Builder. But what is confusing me - I am trying to set up Magento EE + Venia sample data and get no "Venia home page" created automatically. At the same time, all "demo" environments have the same content/images on the home page (e.g. https://develop.pwa-venia.com/ ). Am I missing some sample data extension?